build.gradle

```diff
@@ -19,44 +19,38 @@ buildscript {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
         classpath "com.google.guava:guava:$guavaVersion"
         classpath "com.bertramlabs.plugins:asset-pipeline-gradle:$assetPipelineVersion"
-        classpath 'org.grails.plugins:hibernate5:6.0.4'
+        classpath 'org.grails.plugins:hibernate5:6.1.12'
         classpath "com.google.protobuf:protobuf-java:${protobufVersion}"
         classpath "com.google.protobuf:protobuf-java-util:${protobufVersion}"
-        classpath 'org.grails.plugins:views-gradle:1.0.12'
+        classpath 'org.grails.plugins:views-gradle:1.2.10'
         classpath 'net.linguica.gradle:maven-settings-plugin:0.5'
         classpath "org.fusesource.jansi:jansi:${jansiVersion}"
-        classpath 'org.owasp:dependency-check-gradle:3.2.1'
+        classpath 'org.owasp:dependency-check-gradle:5.3.2.1'
```

Comment:
It is the latest version: https://mvnrepository.com/artifact/org.owasp/dependency-check-gradle

Reply:
you're right, but I have some issue with this version, I'm trying to find out why:
```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/ewelina/transmart-core/build.gradle' line: 34

* What went wrong:
A problem occurred evaluating root project 'transmart-core'.
> Failed to apply plugin [id 'org.owasp.dependencycheck']
   > Could not create task of type 'Analyze'.
```

Reply:
This issue is there only when running the application in development mode (using `grails run-app`), it works fine when running as a .jar. I still haven't found a fix for this.

Reply:
For me the plugin fails with gradle 4, but works with gradle 3.5.1. Weird. I never use `grails run-app` actually.

Reply:
Its does work for me with `grails run-app` and `gradle bootRun` with Grails 3.3.11, Gradle 4.10.3 and Java 8. Could you check the versions?

Reply:
I have the same versions for all and the gradle command works for me as well. It's only `grails run-app` that fails, but if it works for you, it should be fine, maybe it's just my local issue then.